lineapy/utils/config.py

```diff
@@ -132,16 +136,16 @@ def _set_defaults(self):
         self.safe_get("artifact_storage_dir")
         self.safe_get("customized_annotation_folder")
 
-    def safe_get(self, name) -> Union[Path, str]:
-        def safe_get_folder(name) -> Path:
-            """Return folder as pathlib.Path
+    def safe_get(self, name) -> Union[UPath, str]:
```

Comment:
how does passing AWS credentials work?

Reply:
We don't handle AWS credentials directly. We use exactly the same setup when you run `pd.read_csv('s3://some-bucket/some_csv.csv')` or `df.to_csv` since we are using the file handler directly. It will take the default credential in your `.aws` folder. If you don't want to use the default credential, you can do `lineapy.options.set('storage_options', {some key value})` just like in pandas if you don't want to use default credential, you can do `pd.read_csv('s3://some-bucket/some_csv.csv', storage_options={some key value})`.